packages/powertools-logger/src/Logger.ts

```diff
@@ -129,7 +127,7 @@ const makeLoggerInstance = (logger: Logger) => {
     };
 
     unsafeLogger.processLogItem(
-      unsafeLogger.logLevelThresholds[mappedLogLevel[options.logLevel.label]],
```

Comment:
`unsafeLogger.logLevelThresholds` isn't defined in `@aws-lambda-powertools/logger@2`